libexec/bats-core/bats

```diff
@@ -76,7 +76,7 @@ export BATS_TEST_PATTERN_COMMENT="[[:blank:]]*([^[:blank:]()]+)[[:blank:]]*\(?\)
 export BATS_TEST_FILTER=
 export PATH="$BATS_LIBEXEC:$PATH"
 export BATS_ROOT_PID=$$
-if [[ -z "$TMPDIR" ]]; then
+if [[ -z "${TMPDIR-}" ]]; then
```

Comment:
Should this be `:-` as per https://github.com/bats-core/bats-core/pull/392/files#diff-585433432247165e74657c49237de62aa771151126d24c8ee4aa6f47befc7578R28 (and my preference, I think it's a little safer by default)